File: VB/MyBandedGridView/MyBandedGridViewInfo.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports DevExpress.XtraGrid.Views.BandedGrid.ViewInfo
Imports DevExpress.XtraGrid.Views.BandedGrid

Namespace MyXtraGrid
	Public Class MyBandedGridViewInfo
		Inherits BandedGridViewInfo
		Public Sub New(ByVal gridView As DevExpress.XtraGrid.Views.BandedGrid.BandedGridView)
			MyBase.New(gridView)

		End Sub
		Protected Overrides Function CalcMinRowHeight() As Integer
			Return MyBase.CalcMinRowHeight() + 10
		End Function
	End Class
End Namespace

File: CS/MyBandedGridView/MyBandedGridViewInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using DevExpress.XtraGrid.Views.BandedGrid.ViewInfo;
using DevExpress.XtraGrid.Views.BandedGrid;

namespace MyXtraGrid
{
    public class MyBandedGridViewInfo : BandedGridViewInfo
    {
        public MyBandedGridViewInfo(DevExpress.XtraGrid.Views.BandedGrid.BandedGridView gridView)
            : base(gridView)
        {
            
        }
        protected override int CalcMinRowHeight()
        {
            return base.CalcMinRowHeight() + 10;
        }
    }
}
